src/main/java/org/asciidoc/intellij/lexer/asciidoc.flex

```diff
@@ -376,7 +376,7 @@ LINKEND = "]"
                            return textFormat();
                          }
                        }
-  {LINKSTART} / [^\]]+ {LINKEND} { yybegin(LINKFILE); return AsciiDocTokenTypes.LINKSTART; }
+  {LINKSTART} / [^\[\n]* {LINKTEXT_START} [^\]\n]* {LINKEND} { yybegin(LINKFILE); return AsciiDocTokenTypes.LINKSTART; }
```

Comment:
@bbrenne - I tried to make make the look ahead stricter in two ways:
* without a linktext start "[" the parser would otherwise parse until the end of the file
* an empty line would make the link statement invalid and instead end the block. For now I want to keep new lines completely out of the link statement. I would rather skip highlighting and only highlight things I'm very sure of.